app/assets/javascripts/two-pane.js

```diff
@@ -7,7 +7,7 @@ $.fn.tab = function ( option ) {
   }
 }
 
-$(document).on('click', ".table-two-pane td a[href$='edit']", function(e) {
+$(document).on('click', ".table-two-pane td[class$='display-two-pane'] a", function(e) {
```

Comment:
you can use `td.display-two-pane` as well if you only want the `td`s with that class, `[class$=...]` lookups are much more expensive.